swagger.json

```diff
@@ -1804,6 +1804,42 @@
                 }
             }
         },
+        "/industrial/data/{data_input_id}": {
+            "get": {
+                "tags": [
+                    "Industrial", "Default"
+                ],
+                "summary": "/industrial/data/{data_input_id}",
+                "description": "Fetch history of a data input.",
+                "operationId": "GetDataInput",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    { "$ref": "#/parameters/dataInputIdParam"},
+                    { "$ref": "#/parameters/dataInputStartTimeParam"},
+                    { "$ref": "#/parameters/dataInputEndTimeParam"}
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Data points from this data input",
```

Comment:
The `/industrial/data` endpoint uses the noun `data inputs`. We should be consistent with this endpoint.